### ISSUE TITLE
Server Error (500) on API /workout/:id/log_data

### ISSUE THREAD

**Issue by manto89** [wger-project/wger#1180]
Hi,
I am testing the app and found an issue while investigating a bug with the mobile app (see wger-project/flutter#291) .
The endpoint in object always answers with 500 Internal Server Error.
After investigation it seems related to 
```
wger/manager/api/views.py:106 
```
In method `log_data` the object `Exercise` doesn't have a `workoutlog_set` but `ExerciseBase` does! So it seems like an easy fix. Is there anything I'm missing?
I tried to implement the fix and everything seems to be working, can anyone review my fix?
Thanks for the great app!

**Comment by rolandgeider** [comment]
Hi! good catch!

The reason why this is broken is because we recently did a big refactoring of how the exercises work. There is an `ExerciseBase` which holds the base data and and `Exercise` (which historically had everything in it so it probably should be renamed to something like Translation or something like that ). Previously all the logs and workouts were referencing the Exercise model which had some problems such as e.g. the workout templates were locked to the language the exercises were added in the first place. Now we just use the base for everything which is cleaner.

In this case, it would be better to load the base directly (and make sure that the flutter app also sends the correct ID)
